### DISCUSSION TITLE
Session cookie collision with subdomains

### DISCUSSION THREAD

**Discussion by chuckchick** [symfony/symfony#43242] [Q&A]
using: Symfony 5.3.7 with new authenticators/passport.     app hosts both mysite.com    and   *.mysite.com    subdomains

I'm logged in and the session cookie is set to    .mysite.com 
but as soon as I hit   demo.mysite.com   the same session cookie is overwritten and I'm logged out.

`
framework:
   
    session:
        enabled: true
        handler_id: Symfony\Component\HttpFoundation\Session\Storage\Handler\PdoSessionHandler
        storage_factory_id: session.storage.factory.native
        cookie_secure: auto
        cookie_lifetime: 31536000
        gc_maxlifetime: 31536000
        cookie_samesite: null 
        cookie_domain:   .mysite.com    

`

How can I tell Symfony that when the subdomain loads from a controller route to use the same .mysite.com  cookie that already exists, not issue a new cookie and overwrite the logged in one?

**Comment by lyrixx** [comment]
You can use a different cookie name for both application (or domain)
